OAK-Marketplace-Emotion-Recognition/utils/helpers.py

```diff
@@ -0,0 +1,16 @@
+import skvideo.io
+import os
+
```

Comment:
The file is named `helpers.py` but the only thing it does is just saving the video

Reply:
Probably we can call it just utils.py